updatecli/scripts/check-jdk.sh

```diff
@@ -25,11 +25,22 @@ function get_jdk_download_url() {
       ## JDK17 URLs have an underscore ('_') instead of a plus ('+') in their archive names
       echo "https://github.com/adoptium/temurin17-binaries/releases/download/jdk-${jdk_version}/OpenJDK17U-jdk_${platform}_hotspot_${jdk_version//+/_}";
       return 0;;
-    21*)
-      # JDK version (21+35-ea-beta)
+    19*)
```

Comment:
jdk 19 ?

Reply:
Good catch! Removed in https://github.com/jenkins-infra/jenkins-infra/pull/3113/commits/29cd4d756c40b42af68929c9c558bbe45ecf6193